examples/dti_prediction/README.md

```diff
@@ -0,0 +1,40 @@
+# Drug-Target Interaction: DeepDTA
+
+### 1. Description
+Drug-target interaction is a substantial research area in the field of drug discovery. It refers to predict the binding
+affinity between the given chemical compounds and protein targets. In this example we train a standard DeepDTA model as
+baseline in open BindingDB datasets. BindingDB is a public, web-accessible database of measured binding affinities.
```

Comment:
If this example is just about BindingDB, you could consider using it in the example name, like our cifar.